### DISCUSSION TITLE
Problem installing Seurat with Mac M1 chip and R 4.2

### DISCUSSION THREAD

**Discussion by AnonymousGym** [satijalab/seurat#6099] [Q&A]
Hi, when my Macbook moved from Big Sur to Monterey, and my R version moved from 4.0 to 4.2, I can no longer use my Seurat package. 

Here is the error:

Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/Library/Frameworks/R.framework/Versions/4.2/Resources/library/MASS/libs/MASS.so':
  dlopen(/Library/Frameworks/R.framework/Versions/4.2/Resources/library/MASS/libs/MASS.so, 0x0006): tried: '/Library/Frameworks/R.framework/Versions/4.2/Resources/library/MASS/libs/MASS.so' (mach-o file, but is an incompatible architecture (have 'arm64', need 'x86_64'))

The problem with architecture is weird... I'm trying Rossetta for help. Thanks!

**Comment by monkeyBai96** [comment]
Problem library and re-install Seurat with win10 R 4.2, seurat version 4.1.1

R session aborded
